questionnaire-en.json

```diff
@@ -282,7 +273,7 @@
             },
             "3.c.iii": {
               "correctAnswer": "Yes",
-              "question": "     integrated with other open source such that it may trigger copyleft obligations;",
+              "question": "     Containing modified Open Source;",

```

Comment:
We should swap the question back to be consistent with the prior version for compatibility in the answers.

Reply:
Is returning the order to the previous order a hard requirement? If not, the improved readability without changing the meaning of the questions would appear to have minimal impact. Prior conformant organizations will have answered all of the questions with "yes" irrespective of the order, so their conformance situation will remain unchanged in practice.

Reply:
> Is returning the order to the previous order a hard requirement?

If we renumber, than the questions which were answered "yes" or "no" previously will map to a different question.  When the individual goes back into the application, they will see that they answered yes for something they answered no to previously (or visa-versa) since the answers are mapped to a question number and not to the question text.  If I were answering the questions and came back to the application and something I answered No to was now a Yes (or visa-versa) I would think there was a bug in the application.

@shanecoughlan do you agree?

Reply:
As per the comment below, we'll change the version to 2.1.